src/Symfony/Component/Mailer/Transport.php

```diff
@@ -115,6 +116,14 @@ private function parseDsn(#[\SensitiveParameter] string $dsn, int $offset = 0):
             'roundrobin' => RoundRobinTransport::class,
         ];
 
+        $parsedUrl = parse_url($dsn);
+        parse_str($parsedUrl['query'] ?? '', $query);
+        if (isset($query['retry_period'])) {
+            $retryPeriod = min((int)$query['retry_period'], 60);
+        } else {
+            $retryPeriod = 60;
+        }
```

Comment:
```suggestion
        $retryPeriod = min((int)($query['retry_period'] ?? 60), 60);
```
WDYT about this shorter syntax ?